### DISCUSSION TITLE
Fill Box Shape

### DISCUSSION THREAD

**Discussion by Keyvan-Rostyslav** [DmitriySalnikov/godot_debug_draw_3d#49] [General]
Hi, I'm using your plugin, it's great and saves me a lot of time. How can I fill a Box using DebugDraw3D.DrawBox?

**Comment by DmitriySalnikov** [comment]
At the moment, it is not possible to display filled shapes (not a wireframe, but a regular cube, for example).
But I think it's not that difficult to implement. It remains to figure out how to fill my shapes with triangles...

**Comment by DmitriySalnikov** [comment]
I tried to add solid geometry, but again I ran into sorting problems (it is mainly due to the fact that I use MultiMesh)

https://github.com/user-attachments/assets/310e70ce-ac38-4108-b9b9-a42a595e5915

I also tried using dithering, but it has problems too

https://github.com/user-attachments/assets/6487171e-7409-4954-819a-4d12ffe0d84e

https://github.com/user-attachments/assets/1241ce5a-b961-4b15-8568-7c9fc824df12

With such problems, I would not like to add solid geometry.
